src/popup/router/index.js

```diff
@@ -34,6 +34,10 @@ const router = new VueRouter({
   scrollBehavior: (to, from, savedPosition) => savedPosition || { x: 0, y: 0 },
 });
 
+if (window.IS_DESKTOP) {
+  router.afterEach(() => document.querySelector('#app').scroll(0, 0));
+}
+
```

Comment:
```suggestion
router.afterEach(() => document.querySelector('#app').scroll(0, 0));

```
let's make it behave adaptive, e.g. to behave the same without difference on what was initial window size